src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -279,9 +279,30 @@ public AzureAppConfigurationOptions Connect(Uri endpoint, TokenCredential creden
                 throw new ArgumentNullException(nameof(credential));
             }
 
+            return Connect(new List<Uri>() { endpoint }, credential);
+        }
+
+        /// <summary>
+        /// Connect the provider to Azure App Configuration and their replicas using list of endpoints and token credential.
+        /// </summary>
+        /// <param name="endpoints">The list of endpoints of the Azure App Configuration and its replicas to connect to.</param>
```

Comment:
NIT: ... of an Azure App Configuration store and its replicas ...

Reply:
NIT: ... of an Azure App Configuration **store** and its replicas ...

Reply:
Ah, I missed the **store**. Updated now.